pkg/svc/atlas_patch.go

```diff
@@ -207,21 +226,43 @@ The service-defined string used to identify a page of resources. A null value in
 			}
 			op.Parameters = fixedParams
 
+			customResponseCode := responseCodesMap[on]
+
 			// Wrap responses
 			if op.Responses.StatusCodeResponses != nil {
 				// check if StatusCodeResponses has 201 >= x < 300 then delete 200 and don't go to isNilRef check
 				exists := false
-				for code := range op.Responses.StatusCodeResponses {
-					if code >= 201 && code < 300 {
-						exists = true
+				if customResponseCode != 200 {
+					for code := range op.Responses.StatusCodeResponses {
+						if code >= 201 && code < 300 {
+							exists = true
+						}
+						break
 					}
-					break
 				}
+
+				defaultResponseCode := 200
+				if customResponseCode == 200 {
```

Comment:
It can be added in an `else` statement for the `if` block starting on line 235.

Reply:
Sure, done